src/components/contacComponents/contactSection/ContactSection.jsx

```diff
@@ -1,17 +1,10 @@
 import "./contactSection.css";
+import React from "react";
 
 export default function ContactSection() {
   return (
     <div className="contactSection">
       <div className="titleSide">CONTACT</div>
```

Comment:
ContactSection component seems to be basically a header of sort - it is not technically specific to the "contact" area of the project. Therefore, maybe it would be better to just rename it as `Header` (and pass the text that needs to be displayed in it as props). Then it would be possible to use this component in other places of the app as well. Create a new Header folder for it in `src/components`.